README.md

```diff
@@ -1 +1,19 @@
-# Hello-
\ No newline at end of file
+# 
+# Hello Microverse
+
+## Built With
+
+- html
+- css
+
+
+## Author
+
+👤 **Author**
+
+- GitHub: [@EddyShimwa](https://github.com/EddyShimwa)
+- Twitter: [@eddy56388406](https://twitter.com/eddy56388406)
+- LinkedIn: [Pray Eddy SHIMWA]([https://www.linkedin.com/in/pray-eddy-shimwa-18a6751b6/]
+
+
```

Comment:
- [ ] good job creating your readme file with some information 👏 , but to make it more professional and descriptive, kindly copy all of this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) and try to fill out all of the section that helps describe your project example: Getting Started, Setup& Install, Licence, etc ...